src/platforms/common/data-management/debug-files.mdx

```diff
@@ -211,6 +220,22 @@ that is not required to process minidumps. Most notably, inline functions are
 not declared, such that Sentry is not able to display inline frames in stack
 traces.
 
+### WASM
```

Comment:
I'm wondering if the close of the PlatformSection on line 225 should be above this title on line 223? I'm not seeing the WASM header in the Vercel preview.

Reply:
The idea is that since you're in the WASM section this will show only WASM-related information anyway. I think this is fine, but we should consider to do this for other platforms, too.

Reply:
Yeah, that was the idea. I wanted to avoid the single header on a new level when looking at the wasm specific page.